brick/__main__.py

```diff
@@ -126,12 +127,15 @@ def generate_run_command(cmd, run_flags):
 
 def check_recursive(ctx, target, fun):
     if ctx.parent.params.get('recursive'):
-        targets = [os.path.dirname(x) for x in sorted(glob.glob(f'{target}/**/BUILD.yaml', recursive=True))]
+        start = time.perf_counter()
+        targets = [os.path.dirname(x) for x in sorted(wcmatch.WcMatch(f'{target}', 'BUILD.yaml', 'node_modules', flags=wcmatch.RECURSIVE).match())]
```

Comment:
maybe declare `node_modules in a constant`? Is it possible to declare an array of folders to ignore?

Reply:
Makes sense, and yes I think it should be possible - is there anything specific you're thinking of? :)

Reply:
I was just thinking we could do a FOLDERS_TO_IGNORE constant having as value ['node_modules'] (and then we can add other stuff later on).